include/boost/beast/http/error.hpp

```diff
@@ -153,7 +153,15 @@ enum class error
         a new parser for each message. This can be easily done by
         storing the parser in an boost or std::optional container.
     */
-    stale_parser
+    stale_parser,
```

Comment:
What an ugly error message...

Reply:
Wasn’t me but we can change it

Reply:
The problem is not the name but what it represents. The idea that a parser can be "stale" is repugnant. I think I added this to help address a flaw in the design of the parser. beast::http v2.0 needs to address this.

Reply:
Shall we accept this PR and raise an issue about it with a tag of HTTP2?

Reply:
Shall we accept this PR and raise an issue about it with a tag of HTTP2?

Reply:
Yes the PR is fine, and I don't think we need a new issue for the error because it is already covered  in another tagged issue.